src/trace-agent.js

```diff
@@ -35,8 +36,10 @@ var traceAgent;
 function TraceAgent(config, logger) {
   this.config_ = config;
   this.logger = logger;
+  this.plugins = config && config.plugins ? config.plugins : {};
```

Comment:
If a user has an existing configuration that doesn't have a `plugins` entry in their config, using `config.plugins` could cause an error to be thrown when operating on `config.plugins`.

Do we want the agent to fail-fast or to gracefully handle `plugins` not being defined as specifying not to use any plugins?